src/plugins/htmlVideoPlayer/plugin.js

```diff
@@ -277,10 +277,6 @@ export class HtmlVideoPlayer {
          * @type {number | null | undefined}
          */
     #currentTime;
-    /**
-         * @type {any | undefined}
-         */
-    #flvPlayer;
```

Comment:
https://github.com/jellyfin/jellyfin-web/blob/c506d0b753df12eacbd227dc14232c0a6c642c0c/src/plugins/htmlVideoPlayer/plugin.js#L411

Reply:
Yeah it turns out it this is actually referenced [here](https://github.com/jellyfin/jellyfin-web/blob/master/src/components/htmlMediaHelper.js#L237) but that must have been missed when it was converted to being a private member.